var/spack/repos/builtin/packages/libsigsegv/patch.new_config_guess

```diff
@@ -113,7 +113,7 @@
  case "${UNAME_MACHINE}:${UNAME_SYSTEM}:${UNAME_RELEASE}:${UNAME_VERSION}" in
      *:NetBSD:*:*)
  	# NetBSD (nbsd) targets should (where applicable) match one or
--	# more of the tupples: *-*-netbsdelf*, *-*-netbsdaout*,
+-	# more of the tuples: *-*-netbsdelf*, *-*-netbsdaout*,
```

Comment:
Can you please remove this file from the changeset? If we fix the spelling in patches, then we will not be able to apply them.